### DISCUSSION TITLE
Meeting Near Me default location?

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1231] [General]
Can you confirm how the Meeting Plugin defines what "Near Me" is? 

When using this feature, the team and I have only received results from meetings near Melbourne (Australia). These team members who tested live in different parts of the state where there are many other meetings nearby - as the results are displaying closest to Melbourne CBD (Downtown).

This makes me think that it's defaulting to the Office address, which is in Melbourne CBD, that we've used in WordPress, and WooCommerce?

I'm a little confused about how this plugin is choosing Melbourne as a default location, so I'd like to hear your thoughts/advice. 

Please let me know if I missed something, or if I need to select to make "Near Me" an active feature.

View the staging site to test: https://staging.al-anon.org.au/find-a-meeting

**Comment by joshreisner** [comment]
hi, the "Near Me" doesn't use a default address, it uses the browser geolocation API to get the user's location, and, if that's not available, it shows an error and no results

two things that might be causing issues: 

1. geolocation is only available when the website protocol is https, so you should force users to use that. i can visit http://staging.al-anon.org.au/find-a-meeting/ and it doesn't redirect me, when that happens, Near Me is not available as an option. we recommend using a plugin or other setting to ensure that visits to the http version of the site are automatically redirected to the https version
<img width="269" alt="Screenshot 2023-10-10 at 7 40 47 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/35cde1f3-a34d-4d0e-941c-ca9550afce2d">


2. there is a site navigation option to "Find Your Closest Meeting" but it sends you to the link to show all meetings. change this to `https://staging.al-anon.org.au/find-a-meeting/?mode=me` which is the "near me" mode
<img width="382" alt="Screenshot 2023-10-10 at 7 37 03 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/f85477ae-6757-45c7-bfea-ec3e7fb0f194">
